driver/csiplugin/scale_test.go

```diff
@@ -0,0 +1,319 @@
+package scale_test
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+
+	scale "github.com/IBM/ibm-spectrum-scale-csi/driver/csiplugin"
+	"github.com/IBM/ibm-spectrum-scale-csi/driver/csiplugin/connectors"
+	"github.com/IBM/ibm-spectrum-scale-csi/driver/csiplugin/settings"
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"golang.org/x/net/context"
+
+	mock_connectors "github.com/IBM/ibm-spectrum-scale-csi/driver/csiplugin/mock_scale/connectors"
+
+	"github.com/golang/glog"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("CSI Scale Unit Testing", func() {
+	Context("For Controller Expand Volume", func() {
+		var (
+			mockCtrl          *gomock.Controller
+			mockConnectors    *mock_connectors.MockSpectrumScaleConnector
+			mockGetConnectors *mock_connectors.MockGetSpectrumScaleConnectorInt
+			fileset           *connectors.Fileset_v2
+			driver            *scale.ScaleDriver
+			driverName        = flag.String("drivername", "spectrumscale.csi.ibm.com", "name of the driver")
+			nodeID            = flag.String("nodeid", "", "node id")
+			vendorVersion     = "2.8.0"
+			scaleConfig       settings.ScaleSettingsConfigMap
+		)
+		scaleConfig = settings.ScaleSettingsConfigMap{
+			Clusters: []settings.Clusters{
+				{ID: "18359298820404492091",
+					Primary: settings.Primary{
+						PrimaryFSDep: "",
+						PrimaryFs:    "fs1", PrimaryFset: "", PrimaryCid: "", InodeLimitDep: "", InodeLimits: "", RemoteCluster: "", PrimaryFSMount: "", PrimaryFsetLink: "", SymlinkAbsolutePath: "", SymlinkRelativePath: "",
```

Comment:
Try to get config from a common file for testing as well. otherwise it would be difficult every time to edit code for config changes.

Reply:
Moved to common config.go file